### PR TITLE
Make linters fix by default

### PR DESCRIPTION
**Pull request by poneill**

### CODE REVIEW

**Comment by gkucsko** [comment]
hm, fixing pytoml makes sense (though i was pretty sure it was using it by default anyways)
but what's the idea behind removing diff and check only?

**Comment by poneill** [comment]
> hm, fixing pytoml makes sense (though i was pretty sure it was using it by default anyways)
> but what's the idea behind removing diff and check only?

oops yeah you're right about the black config thing, turns out I was just confused by a venv issue.

Seems like if black or isort catch errors, there's really no case where you wouldn't want to fix them.  But we could just have a `--fix` flag instead if you'd rather.

**Comment by gkucsko** [comment]
yeah but does that work in CI? don't think the bot can run tests and just add the fix to the commit...

**Comment by poneill** [comment]
> yeah but does that work in CI? don't think the bot can run tests and just add the fix to the commit...

No, definitely not, it's just for local development.  I'll just add a --fix flag to preserve compatibility.

File: scripts/lint.sh [hunk 1]
```diff
@@ -112,13 +118,23 @@ if [ "$run_fast_linters" -eq 1 ]; then
     echo -e "\n*** End of copyright line check run; exit: $copyright_line_check_exit_code ***\n"
 
     echo -e '*** Running isort... ***\n'
-    isort --check-only --settings-path=setup.cfg --diff --recursive $lintable_locations
```
Comment (poneill) [inline scripts/lint.sh:115]: the --recursive flag is enabled by default since isort 5.0.0, removing it quiets a deprecation warning.

**Review by gkucsko — APPROVED** [review]
lgtm

**Pull request merged**